app/src/components/main/grid/gridBody/gridBody.scss

```diff
@@ -46,15 +46,15 @@
   width: 130px;
   height: 7px;
   margin: 0 auto;
-  background-image: url(./img/accordion-toggle.svg);
+  background-image: url(gridRow/img/accordion-toggle.svg);
```

Comment:
background-image: url(./gridRow/img/accordion-toggle.svg);

Reply:
This scss file will be deleted